src/js/components/Layer/__tests__/__snapshots__/Layer-test.js.snap

```diff
@@ -1005,7 +1005,6 @@ exports[`Layer custom theme 1`] = `
   <div
     class="c2"
     data-g-portal-id="0"
-    elevation="large"
```

Comment:
Is this line removal from snapshot due to using Layer `plain` on the test?

Reply:
This line removal is due to an existing behavior where elevation was leaking to the DOM but shouldn't.

We did not have an existing test with elevation + plain combination. I can add one to this PR for future.

Reply:
Done.